libcalico-go/lib/apis/crd.projectcalico.org/v1/clusterinformation_types.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // ClusterInformation contains the cluster specific information.
 // +k8s:openapi-gen=true
-// +kubebuilder:resource:scope=Cluster
+// +kubebuilder:resource:scope=Cluster,path=clusterinformations
```

Comment:
The `path` option overwrites the built-in pluralization rules (no `s` after `information`).